fastapi-sample/schema.py

```diff
@@ -0,0 +1,108 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+# json読み込み準備
+import json
+with open('./sample.json', 'r') as temp:
+    sample_data = json.load(temp)
+
+
+
+# モックを使ってデータ処理以下
+
+
+# 型チェック用の定義
+from pydantic import BaseModel
+from typing import List
+
+class Item(BaseModel):
+    Message: str
+
+class Items(BaseModel):
+    data:List[Item]
+
+# データの取得
+
+@app.get("/items",response_model=Items)
+async def get_messages():
+    """
+        型チェック(Items)をしてデータを返す
+
+        http://localhost:8000/items
+    """
+    return sample_data
+
+
+@app.get("/item/{id}",response_model=Item)
```

Comment:
```suggestion
@app.get("/items/{id}",response_model=Item)
```
ハンズオンでは言及されていないかもしれませんが、`.../items/{item_id}`となるような設計がSGではセオリーでございます！